echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/pagerduty/PagerDutyNotificationService.groovy

```diff
@@ -82,24 +81,13 @@ class PagerDutyNotificationService implements NotificationService {
         } else {
           pdErrors.put(serviceKey, response.message)
         }
-      } catch (RetrofitError error) {
-        String errorMessage = error.response.reason
-        TypedInput responseBody = error.response.getBody()
-        if (responseBody != null) {
-          PagerDutyErrorResponseBody errorResponse = mapper.readValue(
-            new String(((TypedByteArray)responseBody).getBytes()),
-            PagerDutyErrorResponseBody
-          )
-          if (errorResponse.errors && errorResponse.errors.size() > 0) {
-            errorMessage = errorResponse.errors.join(", ")
-          }
-        }
+      } catch (SpinnakerServerException e){
         log.error('Failed to send page {} {} {}',
           kv('serviceKey', serviceKey), kv('message',
           notification.additionalContext.message),
-          kv('error', errorMessage)
+          kv('error', e.message)
```

Comment:
We're losing a bit of functionality here.  With RetrofitError, errorMessage looked at the `errors` field in the response body.  Now we're only using e.message.  SpinnakerHttpException's message looks at the `message` field, not `errors`.  Seems reasonable to at least try to grab the `errors` property from SpinnakerHttpException's responseBody map.